documentsManager/migrations/0001_initial.py

```diff
@@ -53,13 +57,16 @@ class Migration(migrations.Migration):
         migrations.CreateModel(
             name='TextDoc',
             fields=[
-                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('id', models.AutoField(auto_created=True,
+                                        primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
Correct PEP8 format should be 
         ('id', models.AutoField(
                                                   auto_created=True,
                                                   primary_key=True, 
                                                   serialize=False, 
                                                   verbose_name='ID',
                     )
              ),

Reply:
Todos los code smells son en el archivo de migrations que crea Django. 
Por ahora no los voy a corregir, porque en el proximo PR cuando se creen nuevos van a volver a tener el formato incorrecto.